_posts/blog/2018-19-10-Reseña-Digital.md

```diff
@@ -0,0 +1,31 @@
+
+Es provocador porque rompe con los olvidos dirigidos y recuerdos silenciados, causados por instituciones de poder asociadas a la dominación colonial. Sin embargo, “_los archivos pueden liberar, pero también pueden encerrar, silenciar o decepcionar”_ *1(Weld 2014, 237), para este caso en particular el rol del archivo ha sido liberar, en tanto hace visible documentación relevante para el rescate de la memoria a través de las historias de vidas de familias mapuche. Por lo tanto, la apertura de las 12 familias mapuche más allá de conocer las experiencias, cumplen un rol de empoderamiento a nivel individual y a nivel colectivo, principalmente en las nuevas generaciones, dado que son insumos claves para comprender los fenómenos de migración, las condiciones sociales y políticas en un contexto de despojo forzado.
+
+Constituye un reto para las futuras investigaciones respecto de Pueblos Indígenas en Chile y el Pueblo Mapuche en particular, porque a partir del archivo digital es necesario compatibilizar la recuperación de archivos y tecnología, en tanto estos avanzan en paralelo. La evidencia a partir del “_Archivo Mapuche Warria Mew. Waria ñi tukulpan zugu/ Ampliando los Horizontes de la Memoria”_ así lo demuestra. Otro reto, involucra a los integrantes de pueblos indígenas para reflexionar más allá de las reivindicaciones por los derechos territoriales, sino también el patrimonio cultural e intelectual. 
+
+Finalmente “_Mapuche Warria Mew. Waria ñi tukulpan zugu/ Ampliando los Horizontes de la Memoria”_ es uno de los escasos archivos digitales vinculados a las historias y memorias mapuche realizados por investigadores pertenecientes al mismo pueblo. Es un aporte no solo para los Pueblos Indígenas, sino también para la sociedad en general. Es un material de fácil acceso para instruirse sobre una de las múltiples realidades del Pueblo Mapuche en la actualidad. Por lo tanto, puede ser el inicio para comenzar a desmantelar otros olvidos dirigidos y recuerdos silenciados. 
+
+*1 _“Archives can liberate, but they can also enclose, silence, or disappoint.”_ Frase original, la traduccion es del autor.  
+
+Palabras en Mapuzugün:
```

Comment:
Se puede usar markdown para dar formato a los subtítulos.
 http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/